src/app/qgsmapsavedialog.cpp

```diff
@@ -459,3 +460,9 @@ void QgsMapSaveDialog::accepted()
     }
   }
 }
+
+void QgsMapSaveDialog::showHelp()
+{
+  // description needs to be written and i'm not sure of the place yet
```

Comment:
Is this maybe better left as an issue on the tracker than in here?
I'm afraid that this comment will still be here in 10 years and no one knows why.

I think comments in the code should mainly help a reader understand the code or be very precise todos.

This also applies to all the `//to be precised` comments